lib/rules/no-dynamic-id-classes.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * @fileoverview Don't use classes as an identifier
+ * @author Sandeep Baldawa
+ */
+'use strict'
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Use data-* attributes to provide context to your selectors and insulate them from CSS or JS changes https://docs.cypress.io/guides/references/best-practices.html#Selecting-Elements',
+      category: 'Possible Errors',
+      recommended: true,
```

Comment:
I know this doesn't do anything, but should be `false` here

Reply:
Thanks, changed